src/StatusQ/Components/StatusListItem.qml

```diff
@@ -83,6 +86,9 @@ Rectangle {
 
     property list<Item> components
 
+    signal clicked(string itemId)
```

Comment:
This breaks `StatusNavigationListItem` because it's a `StatusListItem` as well and introduces its own `clicked` handler.